CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
-# Welcome! Thank you for interest in contributing to Akka!
+# Welcome! Thank you for interest in contributing to Pekko!
 
 We follow the standard GitHub [fork & pull](https://help.github.com/articles/using-pull-requests/#fork--pull) approach to pull requests. Just fork the official repo, develop in a branch, and submit a PR!
 
 You're always welcome to submit your PR straight away and start the discussion (without reading the rest of this wonderful doc or the README.md). The goal of these notes is to make your experience contributing to Akka as smooth and pleasant as possible. We're happy to guide you through the process once you've submitted your PR.
 
-## The Akka Community
+## The Pekko Community
 
 If you have questions about the contribution process or discuss specific issues, please visit the [akka/dev Gitter chat](https://gitter.im/akka/dev).
```

Comment:
we don't have gitter - can you remove that? - maybe suggest https://github.com/apache/incubator-pekko/discussions

Reply:
Reminder to lean to 'Apache Pekko' name wise.

Reply:
@hen Do you suggest using "Apache Pekko" everywhere where "Pekko" is used? That might feel a little "non-smooth". I can start with "Apache Pekko" the first time, and then use Pekko the rest of the places. What do you think?

@pjfanning /others: any opinion on this?

Reply:
I believe the convention would be to use "Apache Pekko" initially, then use "Pekko" thereafter. I don't think it makes sense to use "Apache Pekko" everywhere, but it might make sense to use either on section headings or on the first usage of Pekko in the section, since someone might start reading the document there.

Reply:
Done as mentioned above.

Reply:
+1. First use with the full name, then get more casual for the rest of a document.